translator/translate/otel/processor/awsappsignals/testdata/config_k8s.yaml

```diff
@@ -0,0 +1,3 @@
+resolvers:
```

Comment:
we still need to add a config value for cluster name here correct? (separate pr?)